src/main/java/uk/gov/ons/census/action/messaging/CaseAndUacReceiver.java

```diff
@@ -51,15 +57,34 @@ public void receiveEvent(ResponseManagementEvent responseManagementEvent) {
         fulfilmentRequestService.processEvent(
             responseManagementEvent.getPayload().getFulfilmentRequest(), caze, actionType);
       }
-    } else if (eventType == EventType.UAC_UPDATED) {
+      return;
+    }
+
+    if (eventType == EventType.UAC_UPDATED) {
       processUacUpdated(responseManagementEvent.getPayload().getUac());
-    } else if (eventType == EventType.CASE_UPDATED) {
+      return;
+    }
+
+    if (eventType == EventType.CASE_UPDATED) {
       processCaseUpdatedEvent(responseManagementEvent.getPayload().getCollectionCase());
-    } else {
-      // This code can't be reached because under the class structure the EventType is limited to
-      // enums at this point?
-      throw new RuntimeException(String.format("Unexpected event type '%s'", eventType));
+      return;
+    }
+    // This code can't be reached because under the class structure the EventType is limited to
```

Comment:
Change or remove this comment, don't leave it as a question.  Maybe something like:
I'd just remove it